azext_iot/_help.py

```diff
@@ -576,6 +576,49 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Download, upload or migrate the state of an IoT Hub.
+"""
+
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Saves the configuration and device information of an IoT Hub to a file.
+    examples:
+    - name: Export the devices in the specified hub to the specified file.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename}
+"""
+
+helps[
+    "iot hub state import"
+] = """
+    type: command
+    short-summary: Uploads the configuration and device information from a file to an IoT Hub.
```

Comment:
In this import command and migrate command descriptions (as well as export) the command verb isn't being used in the summary description. If 'upload' is a more clear action compared to import then the command action can be renamed to upload so verbiage is consistent....but I think import/export/migrate are totally fine.

Reply:
Overall point being we should be consistent in terminology with descriptions.